### PR TITLE
common: fuel_release_version param is added

### PR DESCRIPTION
**Pull request by rkhozinov**
Related to fuel-qa changes
https://github.com/openstack/fuel-qa/commit/96de67b04506d12047f29cad3d883753b49a425c

### CODE REVIEW

File: servers/partner-ci/common.yaml [hunk 1]
```diff
@@ -1,10 +1,14 @@
 - parameter:
     name: common-parameters
     parameters:
+      - string:
+          name: FUEL_RELEASE_VERSION
+          default: '9.0'
```
Comment (naumvd95) [inline servers/partner-ci/common.yaml]: seems we need to override that with {fuel-release}?

**Review by naumvd95 — CHANGES_REQUESTED** [review]

File: servers/partner-ci/common.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/common.yaml]: I thought about it. But for that case we should create some sets of plugins jobs for each version of fuel I meant 9.0, 9.1, 9.2

Comment (rkhozinov) [reply servers/partner-ci/common.yaml]: Ideally, we could create
9.1 based on the `fuel-qa/9.1` branch
9.x base on the `fuel-qa/stable/mitaka`

Comment (naumvd95) [reply servers/partner-ci/common.yaml]: and 1 more
there is no 9.0 version, as i know, so default value should be 9.1

Comment (rkhozinov) [reply servers/partner-ci/common.yaml]:
```
 File "/home/jenkins/workspace/9.1.contrail.5.0.custom.dev/plugin_test/fuel-qa/fuelweb_test/tests/base_test_case.py", line 381, in check_fuel_version
    assert_equal(fuel_release, settings.FUEL_RELEASE_VERSION, err_msg)
AssertionError: Fuel version should be `9.2` but it is `9.0`
```

**Comment by rkhozinov** [comment]
@naumvd95 stable/5.0.0 has been created by @ilush
and now we should create separate set of jobs to check the ref https://review.openstack.org/#/c/407129/

**Review by naumvd95 — CHANGES_REQUESTED** [review]
need to check that
do u want to create separate jobs?
or remove current?

**Review by naumvd95 — APPROVED** [review]

File: servers/partner-ci/9.0/contrail.yaml [hunk 2]
```diff
@@ -21,30 +20,17 @@
...
-          description: 'proposed repositories to update ubuntu cluster'
-          default: >
-             deb ${{MIRROR}}/pkgs/ubuntu/ trusty main universe multiverse |
-             deb ${{MIRROR}}/pkgs/ubuntu/ trusty-updates main universe multiverse |
-             deb ${{MIRROR}}/pkgs/ubuntu/ trusty-security main universe multiverse
+            CONTRAIL_PLUGIN_PATH=;
+            CONTRAIL_DISTRIBUTION=;
+            CONTRAIL_PLUGIN_PACK_UB_PATH=;
+            MULTIPLE_NETWORKS=;
+            MULTIPLE_NETWORKS_TEMPLATE=;
```
Comment (naumvd95) [inline servers/partner-ci/9.0/contrail.yaml:24]: seems that is useless?

File: servers/partner-ci/9.0/contrail.yaml [hunk 3]
```diff
@@ -21,30 +20,17 @@
...
+            VSRX_CONFIG_PATH=;
 
 - project:
     name: 'predefined_parameters'
     fuel-release: '9.1'
-    plugin-version: '5.0'
+    plugin-version: '5.0.0'
```
Comment (naumvd95) [inline servers/partner-ci/9.0/contrail.yaml]: did u checked, that is no diffs with test-cases?

File: servers/partner-ci/9.0/contrail.yaml [hunk 4]
```diff
@@ -271,7 +253,7 @@
       - '{fuel-release}.{plugin-name}.{plugin-version}.bvt.test.juniper'
 
       - '{regression-name}':
-          testrail_suite: "[9.x][PCE] Fuel plugin contrail v5.0.0 Regression"
+          testrail_suite: "[9.0][PCE] Fuel plugin contrail v5.0.0 Regression"
```
Comment (naumvd95) [inline servers/partner-ci/9.0/contrail.yaml:252]: maybe we need to create separate regression suite or rename current?

**Review by naumvd95 — CHANGES_REQUESTED** [review]

File: servers/partner-ci/common.yaml [hunk 5]
```diff
@@ -347,7 +347,11 @@
                 ${SNAPSHOTS_URL/SNAPSHOTS_ID/$SNAPSHOTS_ID}
                 snapshots_id=$(grep "^CUSTOM_VERSION=" < snapshots.params | cut -d= -f2 | cut -d'#' -f2)
                 echo "snapshot_id_number=" $snapshots_id
+<<<<<<< HEAD
                 python "${CONV_SNAP_PATH}"
+=======
```
Comment (naumvd95) [inline servers/partner-ci/common.yaml]: merge conflict!!!!!

**Review by naumvd95 — CHANGES_REQUESTED** [review]

File: servers/partner-ci/9.0/contrail.yaml (hunk 2 above)
Comment (rkhozinov) [reply servers/partner-ci/9.0/contrail.yaml:24]: they are used in tests, let's save it for custom

File: servers/partner-ci/9.0/contrail.yaml (hunk 3 above)
Comment (rkhozinov) [reply servers/partner-ci/9.0/contrail.yaml]: is it used in tests?

File: servers/partner-ci/9.0/contrail.yaml (hunk 4 above)
Comment (rkhozinov) [reply servers/partner-ci/9.0/contrail.yaml:252]: we'll rename current

**Review by naumvd95 — APPROVED** [review]

File: tox.ini [hunk 6]
```diff
@@ -33,6 +33,33 @@ commands =
    jenkins-jobs --conf {env:CONFIG} test {env:SERVERS}/7.0/vcenter.yaml:{env:COMMON}   -o {env:OUTPUT}
```
Comment (rkhozinov) [inline tox.ini:25]: Remove old

Comment (rkhozinov) [reply tox.ini:25]: Also need to add pip requirements

File: tox.ini [hunk 7]
```diff
@@ -33,7 +25,7 @@ commands =
...
 
 
-[testenv:partner-ci]
+[testenv]
 #deps = -r{toxinidir}/conf/requirements.txt
```
Comment (rkhozinov) [inline tox.ini]: restore that.
get reqs from the jenkins-jobs-infra

**Pull request merged**